autobahn/logger.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+
+def make_logger(logger_type):
+    if logger_type == "twisted":
+        # If we've been asked for the Twisted logger, try and get the new one
+        try:
+            from twisted.logger import Logger
+            return PassthroughLogger(Logger())
+        except ImportError:
+            pass
+
+    from logging import getLogger
+    return PassthroughLogger(getLogger())
+
```

Comment:
Why do we need the pass through logger instead of just returning the logger instance?